src/main/java/it/pn/frontend/e2e/pages/destinatario/DestinatarioPage.java

```diff
@@ -58,5 +68,34 @@ public boolean isDateBoxInvalid(){
         return invalidBoxDate;
     }
 
+    public void clickButtonNotificheOnSideMenu(String nomeDelegante){
+        logger.info("verifica bottone notifiche nel layout");
+
+        getWebDriverWait(10).until(ExpectedConditions.visibilityOf(this.sideItemNotificheButton));
+        sideItemNotificheButton.click();
+
+        String id = "side-item-" + nomeDelegante;
+        By buttonNotificheOnSideMenu = By.id(id);
+        getWebDriverWait(10).withMessage("bottone notifiche nel layout non visibile").until(ExpectedConditions.visibilityOfElementLocated(buttonNotificheOnSideMenu));
+        this.js().executeScript("arguments[0].click()", this.element(buttonNotificheOnSideMenu));
 
+    }
+
+    public void clickSulDettaglioNotificaDelegante(){
+        WebElement singolaNotificaDelegante = listaNotificheDelegante.get(0);
+        getWebDriverWait(10).withMessage("la singola notifica del delegante non è visibile").until(ExpectedConditions.visibilityOf(singolaNotificaDelegante));
+        logger.info("Si clicca sulla notifica del delegante");
+        singolaNotificaDelegante.click();
+    }
```

Comment:
This should indicate that you're clicking the first notification

Reply:
is correct, test need views detai of notification

Reply:
done